app/helpers/remote_execution_helper.rb

```diff
@@ -116,7 +116,7 @@ def job_invocation_task_buttons(task)
                          :class => 'btn btn-default',
                          :title => _('See the last task details'))
     end
-    if authorized_for(:permission => :edit_foreman_tasks, :auth_object => task, :authorizer => task_authorizer)
+    if authorized_for(:permission => :cancel_job_invocations, :auth_object => job_invocation)
       buttons << link_to(_('Cancel Job'), cancel_foreman_tasks_task_path(task),
```

Comment:
It doesn't work as expected: I had an user that had only rex manager rigths: I got an error

```
Please request one of the required permissions listed below from a Foreman administrator:

    edit_foreman_tasks

```
We probably need a separate end-point for the job cancellation, and not use the task end point for it.